pages/dev/gas-receiver.md

```diff
@@ -0,0 +1,134 @@
+# Axelar Gas Receiver
+
+The Axelar Network only validates that a contract call was made from a certain source chain to another destination chain. Execution of the actual contract call is up to the application. We also provide a set of services that can automatically exeute the contract calls on the destination chain by receiving funds on the source chain.
+
+## Introduction
+
+An application that wants Axelar to automatically execute contract calls on the destination chain needs to do four things:
+
+- Estimate the `gasLimit` that the contract call will require on the destionation chain.
+- Query our API to get the `sourceTokenPrice` of the desired token that gas will be payed in, as well as `destinationGasPrice` and `destinationTokenPrice`, the price of the native token, on the destination chain.
+- Calcualate the amount of token to be payed as `gasLimit * destinationGasPrice * gestinationTokenPrice / sourceTokenPrice`.
+- Pay our `AxelarGasReceiver` smart contract on the source chain that amount. This can be done by the application's smart contracts so no additional transactions are required (except maybe approval in case gas is payed in non-native tokens).
+
+Our service does the following:
+
+- Monitors `AxelarGasReceiver` for receipts of payment, and gets the amount payed as `amountPayed`.
+- Matches those to contract calls.
+- Queries our API to get the `sourceTokenPrice` of the token that gas was payed in, as well as `destinationGasPrice` and `destinationTokenPrice`, the price of the native token, on the destination chain.
+- Calcualate the `gasLimit` as `amountPayed * sourceTokenPrice / (destinationGasPrice * gestinationTokenPrice)`.
```

Comment:
`amountPayed` really should be `amountPaid`, but that requires a change to the gas receiver itself, I am guessing.